### PR TITLE
Update Contributing Guidelines and Readme

### PR DESCRIPTION
**Pull request by Julian-O**
### Contributing Guidelines

* Refer to Kivy for the heavy-lifting, and discuss the *differences* with Kivy.
* Made wording more consistent with other sources.
* Removed empty promise of a release every 4 weeks. That isn't a sustainable rate.
* Very mild copy-edit.

### ReadMe

* Refer to Kivy for details of support, rather than repeating.
* Refer to p4a Contributing Guidelines, rather than repeating
* Refer to Quickstart rather than repeating
* Increased space for all the current sponsors and provisioned a far way into the future.
   * Could have used an SVG here, but the comment promised we would include a link :-(

### CODE REVIEW

File: README.md
Comment (AndreMiras) [inline README.md:70]: I'm wondering if we want to keep the history section.
On one hand it may help core contributors to understand some technical choices, on the other hand I'm not sure the README.md is the right place

**Review by AndreMiras — APPROVED** [review]
Left a few easy fix comments, but overall looks good to me

File: README.md
Comment (Julian-O) [reply README.md:30]: Good idea. Done.

File: README.md
Comment (Julian-O) [reply README.md:49]: Done.

File: README.md
Comment (Julian-O) [reply README.md:70]: I understand the dilemma, but there is an easy solution. The Contributing Guidelines are already on the operating table! I have moved the history section into the Contributing Guidelines where it is right there for the people about to contribute.

(If we ever created an "Technical Architecture" section, it might do better there, but this is the best place for now.)

**Comment by Julian-O** [comment]
Thanks for the review, @AndreMiras. Corrections now submitted.

**Review by AndreMiras — APPROVED** [review]

**Comment by Julian-O** [comment]
Some comments on the Kivy/kivy equivalent to this PR may impact this one. Please don't merge for a few days while it is resolved.

**Review by misl6 — CHANGES_REQUESTED** [review]
I see a lot of unrelated commits have been added to this branch.

Can you please rebase instead?

**Comment by Julian-O** [comment]
> I see a lot of unrelated commits have been added to this branch.

I have some elaborate theories about why Git personally hates me so much.

Fixed.

**Review by misl6 — CHANGES_REQUESTED** [review]
Hi @Julian-O ,

Is all almost fine, except for the APK/AAB/AAR part.
I've left a comment on README.md, but a similar issue is present on `doc/source/index.rst`.

Feel free to ping me on Discord if you need some further explanation.

File: doc/source/troubleshooting.rst
Comment (Julian-O) [reply doc/source/troubleshooting.rst:5]: I dunno, I kind of like it. :-)

Fixed.

File: README.md
Comment (Julian-O) [reply README.md:15]: Substantially rewrote these sections

**Review by misl6 — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**
